.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug report
+description: Report an unexpected behavior
+labels: ["kind: bug", "needs: triage"]
```

Comment:
I assume people with rights on the repo can change the labels in the creation page, is that correct?

Reply:
Yes.